packages/widgets/src/blocks/legacy-widget/edit/preview.js

```diff
@@ -11,9 +11,19 @@ import { addQueryArgs } from '@wordpress/url';
 import { useState } from '@wordpress/element';
 import { Placeholder, Spinner, Disabled } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
 
 export default function Preview( { idBase, instance, isVisible } ) {
 	const [ isLoaded, setIsLoaded ] = useState( false );
+	const settings = useSelect( ( select ) => {
+		return select( blockEditorStore ).getSettings();
+	}, [] );
+
+	const widgetPreviewUrl = settings.wpAbsoluteAdminUrl + 'widgets.php';
```

Comment:
Nitpick: If `wpAbsoluteAdminUrl` for some reasons isn't defined then the url will be broken. I'm not sure if there's any case where it will not be defined though. But doesn't hurt to add some guard around it?

Reply:
Good catch!
Fixed in 36370d9820206909f8821e746d56c2f3af25f8c2